Code/GraphMol/MolDraw2D/MolDraw2DUtils.cpp

```diff
@@ -481,6 +481,7 @@ void drawMolACS1996(MolDraw2D &drawer, const ROMol &mol,
 
 // ****************************************************************************
 void setACS1996Options(MolDrawOptions &opts, double meanBondLen) {
+  PRECONDITION(meanBondLen > 0.0, "mean bond length <= 0 for ACS1996 mode")
```

Comment:
I think this would be better as a ValueError than a PRECONDITION (which generally indicates a catastrophic programming failure, not bad input)